### PR TITLE
Fixes flakiness in test_ls

### PR DESCRIPTION
**Pull request by dposada**

### CODE REVIEW

**Comment by shamsimam** [comment]
Good to merge?

File: integration/tests/cook/test_cli.py [hunk 1]
```diff
@@ -771,7 +771,7 @@ def entry(name):
         self.logger.debug(entries)
         foo = entry('foo')
         self.assertEqual('drwxr-xr-x', foo['mode'])
-        self.assertEqual(2, foo['nlink'])
+        self.assertGreaterEqual(2, foo['nlink'])
```
Comment (DaoWen) [inline integration/tests/cook/test_cli.py]: This should be `assertLessEqual` if `foo['nlink']` is sometimes greater than 2.

Comment (dposada) [reply integration/tests/cook/test_cli.py]: Fixed, thanks

**Comment by dposada** [comment]
@DaoWen any other feedback on this?

**Pull request merged**